components/content_settings/core/browser/content_settings_util.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_CONTENT_SETTINGS_CORE_BROWSER_CONTENT_SETTINGS_UTIL_H_
+#define BRAVE_COMPONENTS_CONTENT_SETTINGS_CORE_BROWSER_CONTENT_SETTINGS_UTIL_H_
+
+#include <string>
+
+#include "components/content_settings/core/common/content_settings_types.h"
+#include "url/gurl.h"
```

Comment:
forward declaration should be enough?

Reply:
no, it's used as `const GURL&` so you need the header

Reply:
const ref allows the type to be forward declared

Reply:
I guess that would make sense because it's effectively a pointer, but I don't think it's a big deal to have gurl.h here. It's such a common include that it's not going to have any significant impact on rebuilds, etc...